internal/controller/protectedvolumereplicationgrouplist_controller_test.go

```diff
@@ -91,11 +92,64 @@ func protectedVrgListExpectIncludeOnly(protectedVrgList *ramen.ProtectedVolumeRe
 	Expect(protectedVrgList.Status.Items).To(ConsistOf(vrgsExpected))
 }
 
+func matchVRG(expected, got ramen.VolumeReplicationGroup) (found, valid bool, msg string) {
+	if expected.Namespace != got.Namespace || expected.Name != got.Name {
+		return false, false, fmt.Sprintf("VRG %s/%s does not match expected: "+
+			"got: %s/%s",
+			expected.Namespace, expected.Name,
+			got.Namespace, got.Name)
```

Comment:
This message will be generated for all vrgs but one. Does not seem very helpful.